lib/common_test/src/ct_util.erl

```diff
@@ -202,9 +202,7 @@ do_start(Parent, Mode, LogDir, Verbosity) ->
 	    Parent ! {self(),started};
 	{fail,CTHReason} ->
 	    ErrorInfo = if is_atom(CTHReason) ->
```

Comment:
CTHReason is always a string, so you can remove the `if` statement and use `CTHReason` directly instead of `ErrorInfo`.